apps/portals/my-pages/src/components/Notifications/NotificationButton.tsx

```diff
@@ -28,7 +28,7 @@ const NotificationButton = ({
   const [hasMarkedLocally, setHasMarkedLocally] = useState(false)
   const [markAllAsSeen] = useMarkAllNotificationsAsSeenMutation()
   const { width } = useWindowSize()
-  const isMobile = width < theme.breakpoints.md
+  const isMobile = width < theme.breakpoints.lg
```

Comment:
Nitpick: Should this be `isTablet` rather?

In other files you are using `isMobile` when `width < theme.breakpoints.md` and in `NotificationMenu.tsx` you specify 'isTablet' when ` width < theme.breakpoints.lg`.

Reply:
Makes sense 🫡